rfcs/0006-runtime-references-interface.md

```diff
@@ -44,6 +45,11 @@ its builtin hash scanning and just use the file. Both files are
 newline-separated lists of store paths. Parse errors in the
 `runtime-references` file, or non-existent store paths, fail the build.
 
+To avoid duplicating the default reference scanning behavior in Nix,
+we could also make a tiny tool part of Nix itself that wraps the
+`scanForReferences` API, and then expose that in the `<nix>` namespace
+so tooling can get the full path to that tool.
```

Comment:
Ah, hmm. Then it might still be hard to share code, right? Last I checked, Nix was just using its standard sink mechanism to scan for references. I have a much murkier understanding of sharing code between Nix and nixpkgs if it's a normal derivation. I'm still mostly concerned about the transition story.